wyvern/feature_store/historical_feature_util.py

```diff
@@ -117,6 +118,100 @@ def build_historical_real_time_feature_requests(
     return result_dict
 
 
+def build_and_merge_realtime_pivot_tables(
```

Comment:
we should create a one pager that explains the logic clearly with examples. we can even get sth together using the stuff we went over to clarify the logic

Reply:
https://docs.wyvern.ai/training_data

Maybe here?

Reply:
good point. i'll write something after i test this